dbfit-java/db2/src/integration-test/resources/install_db2.sh

```diff
@@ -10,7 +10,8 @@ IM="$MODNAME: INFO:"
 EM="$MODNAME: ERROR:"
 WM="$MODNAME: WARNING:"
 
-DB2_INST_ROOT=/var/dbfit
+DBFIT_ROOT=/var/dbfit
+DB2_INST_ROOT=$DBFIT_ROOT
```

Comment:
We're anyway resetting this to `/tmp` later - why not initialize it here directly to `/tmp`?